src/Core/Components/Badge/FluentBadge.razor.cs

```diff
@@ -0,0 +1,106 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components.Utilities;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// The FluentBadge component is a visual indicator that communicates a status or description of an associated component.
+/// It uses short text, color, and icons for quick recognition and is placed near the relavant content.
+/// </summary>
+public partial class FluentBadge : FluentComponentBase
+{
+    /// <summary />
+    public FluentBadge()
+    {
+        Id = Identifier.NewId();
```

Comment:
Why do we have to have an ID? If it's not mandatory for the component to work (e.g. JS or CSS), I wouldn't put it in.